src/spec-node/containerFeatures.ts

```diff
@@ -89,6 +89,10 @@ export async function extendImage(params: DockerResolverParameters, config: Subs
 			params.additionalCacheFroms.forEach(cacheFrom => args.push('--cache-from', cacheFrom));
 		}
 
+		if (params.buildxCacheFrom) {
```

Comment:
Question: How is `buildxCacheFrom` different from the existing `additionalCacheFroms` property?

Reply:
Is not. Fixed.